cmake/test/gtest.cmake

```diff
@@ -389,13 +325,12 @@ if(NOT GMOCK_FOUND)
             endif()
             _install(${ARGN})
           endfunction()
-          add_subdirectory(${gtest_base_dir} ${gtest_lib_dir})
+          add_subdirectory(${base_dir} ${gtest_lib_dir})
           set(_CATKIN_SKIP_INSTALL_RULES FALSE)
           set_target_properties(${gtest_libs} ${gtest_main_libs}
                                 PROPERTIES EXCLUDE_FROM_ALL 1)
```

Comment:
This property also needs to be set for the gmock targets.

Reply:
You are right, I have ammended the commit to include this change.